package/health_daemon/src/glo_bias_monitor.c

```diff
@@ -22,56 +22,25 @@
 #include "health_monitor.h"
 #include "utils.h"
 
+#include "glo_health_context.h"
 #include "glo_bias_monitor.h"
 
 /* these are from fw private, consider moving to libpiski */
 #define MSG_FORWARD_SENDER_ID (0u)
 
-#define GNSS_BIAS_ALERT_RATE_LIMIT (240000) /* ms */
+#define GLO_BIAS_ALERT_RATE_LIMIT (240000) /* ms */
 
 static health_monitor_t *glo_bias_monitor;
 
-static struct glo_bias_ctx_s {
-  bool glo_setting_read_resp;
-  bool glonass_enabled;
-} glo_bias_ctx = { .glo_setting_read_resp = false, .glonass_enabled = false };
-
-static void
-sbp_msg_read_resp_callback(u16 sender_id, u8 len, u8 msg_[], void *ctx)
```

Comment:
Not sure I understand why we can remove this.  Are we querying the same setting in another health monitor?

Reply:
Yeah this is part of the short term fix, which is to create a shared context `glo_health_context` which is effectively managed by the glo obs monitor. I almost merged both modules into one but went with this approach to keep the logic separate.